src/components/congratulations.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+
+
+  <section class="container"> 
```

Comment:
Did you intend to add another `.container` here? The `#app` element already has the `.container-fluid` class